.circleci/config.yml

```diff
@@ -89,7 +89,8 @@ jobs:
           at: ~/addons-scanner-utils
       - run:
           name: authenticate with registry
-          command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/addons-scanner-utils/.npmrc
+          # We don't want to expand the token in this file, npm will do it.
+          command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/addons-scanner-utils/.npmrc
```

Comment:
@willdurand this should also be changed to a single quote, is that fixed in a separate PR?

Reply:
it looks it has been already updated. This is already resolved on master.

Reply:
yes, I fixed it already.. damn I don't know what happened.